src/App.jsx

```diff
@@ -1,31 +1,35 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { getAllData } from './util/index';
+import BookCardMain from '/src/components/BookCard/BookCardMain.jsx';
+import booksData from '/src/data/booksData.js';
 
 const URL = 'http://localhost:8000/api/v1/';
 
 function App() {
-  
-  const [message, setMessage] = useState(''); 
-
-  useEffect(() => {
-
-    (async () => {
-      const myData = await getAllData(URL)
-      setMessage(myData.data);
-    })();
-      
-    return () => {
-      console.log('unmounting');
-    }
-
-  }, []);
-
-  return (
-    <>
-      <h1>{message}</h1>
-    </>
-  );
-
+	const [message, setMessage] = useState('');
+
+	useEffect(() => {
```

Comment:
This does not appear to be doing anything useful. Can we take this out? You can leave the util functions in `util/index.js`, but this useEffect that doesn't query any data should be taken out to avoid confusion.